src/components/tx/modals/NftTransferModal/SendNftForm.tsx

```diff
@@ -177,7 +170,7 @@ const SendNftForm = ({ params, onSubmit }: SendNftFormProps) => {
                     <MenuItem key={item.address + item.id} value={item.id}>
                       <NftMenuItem
                         image={item.imageUri || item.logoUri}
-                        name={item.name || item.tokenName}
+                        name={item.name || (item.tokenName || item.tokenSymbol || '') + item.id}
```

Comment:
Looks good, but this will concat the id w/o any separator. It used to be `Bored Apes #14545`